tests/TestCsvParser.cpp

```diff
@@ -111,7 +111,7 @@ void TestCsvParser::testEmptySimple()
     out << "";
     QVERIFY(parser->parse(file.data()));
     t = parser->getCsvTable();
-    QVERIFY(t.size() == 0);
+    QVERIFY(t.empty());
```

Comment:
These should be isEmpty(), since we are writing Qt.

Reply:
Yah I believe empty() is similar to clear(). Great catch

Reply:
No, empty is not  similar to clear. 
IsEmpty() is equivalent for empty(), but more native for qt, OK.